src/sentry/runner/commands/cleanup.py

```diff
@@ -187,6 +187,7 @@ def is_filtered(model):
         DELETES = [
             (models.EventAttachment, "date_added", "date_added"),
             (replay_models.ReplayRecordingSegment, "date_added", "date_added"),
+            (models.MonitorCheckIn, "date_added", "date_added"),
```

Comment:
can this use the BulkDeleteQuery (L178) instead? are there any child relations that we need to take care of? i don't think so?

Reply:
ahh yes yes, lemme swap it over to bulk query deletes

Reply:
also assuming i don't need the sort order over in bulk query deletes

Reply:
yup 👍🏼